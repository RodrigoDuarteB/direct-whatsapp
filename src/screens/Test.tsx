import React, { FC, useEffect, useState, memo } from 'react'
import { View, StyleSheet, Text, TouchableOpacity, FlatList, ScrollView, Image } from 'react-native'
import Select from '../components/Select'
import { getCountryByCallingCode, getAllCountries } from '../services/Countries.service'
import { Dropdown } from 'react-native-element-dropdown'
import { Country } from '../models/models'
import { globalStyles } from '../styles/globals'
import CountriesService from '../services/CountriesService'
import DeviceCountry from 'react-native-device-country';
import LocalStorageService from '../services/LocalStorageService'

interface IProps {

}

const Test: FC<IProps> = (props) => {
    const [countries, setCountries] = useState<Country[]>([])
    const [country, setCountry] = useState<Country | null>(null)

    useEffect(() => {
        /* LocalStorageService.saveMessage({
            message: 'mensaje de prueba para el local storage service',
            phoneNumber: '78496366',
            country: {
                code: {
                    root: '+5',
                    suffix: '91'
                },
                flag: 'https://flagcdn.com/w320/bo.png',
                name: 'Bolivia' 
            }
        })
        .then(res => console.log(res)) */
    }, [])

    return (
        <View style={{padding: 15}}>
            <Text style={{ backgroundColor: 'pink'}}>Holaa</Text>
        </View>
    )
}

const styles = StyleSheet.create({
    text: {
        color: 'black',
        backgroundColor: 'pink',
        width: 50
    },
    select: {
        paddingVertical: 10,
        paddingHorizontal: 15
    }
})

export default Test