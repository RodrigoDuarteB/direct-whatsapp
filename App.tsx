import { NavigationContainer } from '@react-navigation/native'
import { createNativeStackNavigator } from '@react-navigation/native-stack'
import React, { FC } from 'react'
import History from './src/screens/History'
import Home from './src/screens/Home'
import Test from './src/screens/Test'

interface IProps {

}

const Stack = createNativeStackNavigator()

const App: FC<IProps> = (props) => {
    return (
        <NavigationContainer>
            <Stack.Navigator 
                initialRouteName='Home' 
                screenOptions={{
                    headerShown: false
                }}
            >
                <Stack.Screen name='Home' component={Home}/>
                <Stack.Screen name='History' component={History}/>
                <Stack.Screen name='Test' component={Test}/>
            </Stack.Navigator>
        </NavigationContainer>
    )
}

export default App