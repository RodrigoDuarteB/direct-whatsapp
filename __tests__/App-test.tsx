/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../App';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import Test from '../src/screens/Test';

it('renders correctly', () => {
  const tree = renderer.create(<Test />).toJSON();
  expect(tree).toMatchSnapshot();
});
