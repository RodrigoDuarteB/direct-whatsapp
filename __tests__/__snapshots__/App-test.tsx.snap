// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "padding": 15,
    }
  }
>
  <Text
    style={
      {
        "backgroundColor": "pink",
      }
    }
  >
    Holaa
  </Text>
</View>
`;
